crates/task-impls/src/events.rs

```diff
@@ -44,6 +44,10 @@ pub enum HotShotEvent<TYPES: NodeType> {
     DACValidated(DACertificate<TYPES>),
     /// Send a quorum proposal to the network; emitted by the leader in the consensus task
     QuorumProposalSend(Proposal<TYPES, QuorumProposal<TYPES>>, TYPES::SignatureKey),
+    /// Dummy quorum proposal to test if the quorum proposal dependency works.
+    DummyQuorumProposalSend(Proposal<TYPES, QuorumProposal<TYPES>>, TYPES::SignatureKey),
```

Comment:
I think I might delete this before this merges and instead just send the real thing since I'll be adding testing next.